micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/MnJobExecutor.java

```diff
@@ -15,19 +15,52 @@
  */
 package info.novatec.micronaut.camunda.bpm.feature;
 
-import org.camunda.bpm.engine.impl.jobexecutor.DefaultJobExecutor;
-
+import io.micronaut.scheduling.TaskExecutors;
+import io.micronaut.scheduling.annotation.ExecuteOn;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.RejectedExecutionException;
+import javax.inject.Named;
 import javax.inject.Singleton;
+import org.camunda.bpm.engine.impl.ProcessEngineImpl;
+import org.camunda.bpm.engine.impl.jobexecutor.JobExecutor;
 
 /**
  * Micronaut specific implementation of the {@link org.camunda.bpm.engine.impl.jobexecutor.JobExecutor}
  *
  * @author Tobias Schäfer
+ * @author Alexander Rolfes
  */
 @Singleton
-public class MnJobExecutor extends DefaultJobExecutor {
+@ExecuteOn(TaskExecutors.IO)
```

Comment:
with one of you last pushes you added `@ExecuteOn(TaskExecutors.IO)`. Is it needed? If yes, it might be easier if we talk ....

Reply:
Sorry, this was a try
I will remove it